packages/blockchain-wallet-v4-frontend/src/modals/CoinifyExchangeData/Create/AcceptTerms/index.js

```diff
@@ -18,7 +18,7 @@ class AcceptTermsContainer extends Component {
   }
 
   componentDidUpdate (prevProps) {
-    if (prevProps.signupError) {
+    if (prevProps.signupError && this.state.busy) {
```

Comment:
@sendmebiscuits I think we should check `this.props.signupError` instead of `prevProps.signupError`otherwise this `if` block will never run. You also need to use eslint-disable and eslint-enable or the linter will complain about setState inside componentDidUpdate. In this case, there shouldn't be any updating loop, I checked.

A good way to test this is to set an incorrect` partnerId` in `config/wallet-options` and then try to create a coinify account. You should see the scenario play out. Looks good otherwise!

Reply:
Ah alright that makes sense! I'm also gonna add the eslint-disable/enable to the other componentDidUpdate and componentDidMount methods in the other files for consistency's sake.

Reply:
Thanks, sounds good to me 👍